server/service/client_trigger.go

```diff
@@ -0,0 +1,50 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// TriggerCronSchedule attempts to trigger an ad-hoc run of the named cron schedule.
+func (c *Client) TriggerCronSchedule(name string) error {
+	verb, path := http.MethodGet, "/api/latest/fleet/trigger"
```

Comment:
I would've expected this to be a POST, since it triggers something within the system and doesn't really retrieve any data.

Reply:
I should've caught this before, sorry!

Reply:
Yeah, I wasn't really sure which verb to use. I noticed that we use GET for [running a live query](https://fleetdm.com/docs/using-fleet/rest-api#run-live-query) and thought maybe that was our established pattern. But it feels weird to me too like you say.

Reply:
But I suppose one distinction is that the live query is returning results

Reply:
I thought we changed this for a POST, did I misunderstand?

Reply:
Good catch, thank you! We did change the endpoint in another [PR](https://github.com/fleetdm/fleet/pull/8888) and it didn't get updated here. I'll push a change and add some more tests :)